appveyor.yml

```diff
@@ -39,7 +45,17 @@ build_script:
 after_build:
 - cmd: move build_msvc\%PLATFORM%\%CONFIGURATION%\*.iobj build_msvc\cache\
 - cmd: move build_msvc\%PLATFORM%\%CONFIGURATION%\*.ipdb build_msvc\cache\
-- cmd: del C:\Users\appveyor\clcache\stats.txt
+- cmd: clcache -z
+before_test:
+- ps:  $conf_ini = (Get-Content([IO.Path]::Combine($env:APPVEYOR_BUILD_FOLDER, "test", "config.ini.in")))
+- ps:  $conf_ini = $conf_ini.Replace("@abs_top_srcdir@", $env:APPVEYOR_BUILD_FOLDER).Replace("@abs_top_builddir@", $env:APPVEYOR_BUILD_FOLDER).Replace("@EXEEXT@", ".exe")
+- ps:  $conf_ini = $conf_ini.Replace("@ENABLE_WALLET_TRUE@", "").Replace("@BUILD_BITCOIN_UTILS_TRUE@", "").Replace("@BUILD_BITCOIND_TRUE@", "").Replace("@ENABLE_ZMQ_TRUE@", "")
+- ps:  $utf8 = New-Object System.Text.UTF8Encoding $false
+- ps:  '[IO.File]::WriteAllLines([IO.Path]::Combine($env:APPVEYOR_BUILD_FOLDER, "test", "config.ini"), $conf_ini, $utf8)'
 test_script:
 - cmd: build_msvc\%PLATFORM%\%CONFIGURATION%\test_bitcoin.exe
+- cmd: build_msvc\%PLATFORM%\%CONFIGURATION%\bench_bitcoin.exe -evals=1 -scaling=0
+- cmd: python test\util\bitcoin-util-test.py
+- cmd: python test\util\rpcauth-test.py
+- cmd: python test\functional\test_runner.py --force --quiet --combinedlogslen=4000 --exclude "wallet_multiwallet,wallet_multiwallet.py --usecli"
```

Comment:
using `ps` here would be better.

Reply:
`ps` on appveyor can't show unicode correctly
https://help.appveyor.com/discussions/problems/1105-utf-8-characters-in-powershell-log

Reply:
So you would prefer to use `ps` if possible? If so, I would change those except 
```
python test\functional\test_runner.py --force --quiet --combinedlogslen=4000 --exclude "wallet_multiwallet,wallet_multiwallet.py --usecli"
```
because of appveyor ps issue.

Reply:
Interesting, yes I think that is a good choice.
Should probably add a short comment about it.

Reply:
ps